tests/food_fun_for_test.cpp

```diff
@@ -142,29 +142,29 @@ TEST_CASE( "fun for cold food", "[fun_for][food][cold]" )
     }
 
     GIVEN( "food that tastes bad, but better when cold" ) {
-        item sports( "sports_drink" );
-        REQUIRE( sports.is_comestible() );
-        int sports_fun = sports.get_comestible_fun();
+        item rehydration( "rehydration_drink" );
```

Comment:
```suggestion
        item sports( "rehydration_drink" );
```
You can simply change the item id in this line, and leave all the rest of the test exactly as it was. There is no need to change the name of the `rehydration` variable or the `rehydration_fun` variable.

Reply:
Alright, thanks for the tip. I probably won;t be changing it, but I'll keep that in mind!

Reply:
Well, you already did change the variable names from `sports` and `sports_fun` to `rehydration` and `rehydration_fun`. What I mean is, you can revert those changes back out of this PR, since they are not necessary for the test to work.

Reply:
I had suggested changing those because it was confusing, at least to me, to have it saying "sports" when it wasn't the sports drink.

Reply:
Understood. This is another reason to use a `test_sports_drink` item here, so it will not be affected by similar changes in the future.